docs/src/common.md

```diff
@@ -75,26 +75,31 @@ The equations of motion for the PCR3BP are
 \end{aligned}
 ```
 
-We define this system of ODEs using the most naive approach
+We define this system of ODEs in a way that allows the use of the [`@taylorize`](@ref)
+macro from `TaylorIntegration.jl`, which for the present example allows
+important speed-ups. For more details about the specifics of the use of [`@taylorize`](@ref),
+see [this section](@ref taylorize).
 ```@example common
-function f(dq, q, param, t)
+using TaylorIntegration
+@taylorize function pcr3bp!(dq, q, param, t)
     local μ = param[1]
-    x, y, px, py = q
-    dq[1] = px + y
-    dq[2] = py - x
-    dq[3] = - (1-μ)*(x-μ)*((x-μ)^2+y^2)^-1.5 - μ*(x+1-μ)*((x+1-μ)^2+y^2)^-1.5 + py
-    dq[4] = - (1-μ)*y    *((x-μ)^2+y^2)^-1.5 - μ*y      *((x+1-μ)^2+y^2)^-1.5 - px
+    local onemμ = 1 - μ
+    x1 = q[1]-μ
+    x1sq = x1^2
+    y = q[2]
+    ysq = y^2
+    r1_1p5 = (x1sq+ysq)^1.5
+    x2 = q[1]+onemμ
+    x2sq = x2^2
+    r2_1p5 = (x2sq+ysq)^1.5
+    dq[1] = q[3] + q[2]
+    dq[2] = q[4] - q[1]
+    dq[3] = -((onemμ*x1)/r1_1p5) - ((μ*x2)/r2_1p5) + q[4]
+    dq[4] = -((onemμ*y )/r1_1p5) - ((μ*y )/r2_1p5) - q[3]
```

Comment:
Not sure if it helps for performance, but i would add one (pair) parenthesis to truly make this evaluation a composition of binary operations, so that it reads as `(a+b)+c` or `a + (b+c)`.

Reply:
Parentheses added 👍